cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -139,14 +138,14 @@ def finalize():
 
 @pytest.mark.tier(2)
 @pytest.mark.ignore_stream("upstream")
-def test_external_database_appliance(request, virtualcenter_provider):
+def test_external_database_appliance(request, virtualcenter_provider, appliance):
     """Tests that one appliance can externally
        connect to the database of another appliance.
 
     Metadata:
         test_flag: replication
     """
-    appl1, appl2 = get_distributed_appliances()
+    appl1, appl2 = get_distributed_appliances(appliance)
```

Comment:
**REQUIRED**: This fn doesn't take an arg

Reply:
It does a couple of lines above ...